DotNetRu.Clients.Portable/ViewModel/SpeakersViewModel.cs

```diff
@@ -70,7 +74,44 @@ orderby speaker.FullName
 		public ICommand LoadSpeakersCommand =>
 			loadSpeakersCommand ?? (loadSpeakersCommand = new Command(async (f) => await ExecuteLoadSpeakersAsync((bool) f)));
 
-		async Task<bool> ExecuteLoadSpeakersAsync(bool force = false)
+	    private static IEnumerable<Speaker> GetSpeakers()
+	    {
+	        var assembly = Assembly.Load(new AssemblyName("DotNetRu.DataStore.Audit"));
```

Comment:
Seems a bit overkill to load an assembly just to get resource file. You can just put the file somewhere and load it.